lib/core/test/unit/Cardano/Wallet/Primitive/FeeSpec.hs

```diff
@@ -170,14 +170,14 @@ spec = do
             , csChngs = [4,1,1]
             })
 
-        -- Fee divvied, dust removed (dust = 1)
+        -- Fee divvied, dust removed (dust = 2)
         feeUnitTest id (FeeFixture
             { fInps = [20,20,20]
             , fOuts = [14,18,19]
             , fChngs = [6,2,1]
             , fUtxo = []
             , fFee = 3
-            , fDust = 1
```

Comment:
Without this fix, the error was

```
  test/unit/Cardano/Wallet/Primitive/FeeSpec.hs:622:5:
  1) Cardano.Wallet.Primitive.Fee, Fee calculation : unit tests, CoinSelection (inps=[20,20,20]outs=[14,18,19]chngs=[6,2,1]), UTxO=[]), fee=3 --> Right (FeeOutput {csInps = [20,20,20], csOuts = [14,18,19], csChngs = [6]})
       expected: Right (FeeOutput {csInps = [20,20,20], csOuts = [14,18,19], csChngs = [6]})
        but got: Right (FeeOutput {csInps = [20,20,20], csOuts = [14,18,19], csChngs = [4,1,1]})
```

which looking at it now makes sense. `1` in change output is allowed when dust threshold is `1`, so this should pass. Changing the threshold to `2` makes the unit test more exciting though.